testing_and_setup/compass/ocean/jigsaw_to_MPAS/build_mesh.py

```diff
@@ -57,13 +63,22 @@ def removeFile(fileName):
 print "running", ' '.join(args)
 subprocess.check_call(args, env=os.environ.copy())
 
-print 'Step 6. Injecting bathymetry'
-args = ['./inject_bathymetry.py',
-        'base_mesh.nc']
-print "running", ' '.join(args)
-subprocess.check_call(args, env=os.environ.copy())
+if cl_args.inject_bathymetry:
+  print 'Step 6. Injecting bathymetry'
+  args = ['./inject_bathymetry.py',
+          'base_mesh.nc']
+  print "running", ' '.join(args)
+  subprocess.check_call(args, env=os.environ.copy())
+
+if cl_args.inject_cullCell > 0.0:
+  print 'Step 7. Injecting cell culling flag'
+  args = ['./inject_cullCell.py',
```

Comment:
@sbrus89, are we sure this is the right name here?  The `cellSeedMask` is related to preservation of flood fill.  So, I'm not sure this is the best name we can use and would recommend something more like `inject_preserve_floodplain.py` to be a little clearer.  This helps as we do more ocean-land interactions because that will be particularly informative for those applications.  

Also, long term we will need some way to make this more prescriptive because "dry" cells are essentially waste and we may also need to set this mask based on some notion of the river / channel thalwegs.

Reply:
I agree, this name doesn't really make sense anymore. I'll go ahead and change it.